bedrock/mozorg/templates/mozorg/home/home-new.html

```diff
@@ -63,8 +65,9 @@
 <header>
     {%- if LANG == 'de' and switch('de-ctd-homepage-promo') -%}
       {% include 'mozorg/home/includes/ctd-promo-de.html'%}
+    {% elif show_pocket_banner %}
+      {% include 'mozorg/home/includes/pocket-pencil-banner.html' %}
```

Comment:
Is this an in-page promo or a dismiss-able banner? If it's supposed to be a banner, then it should not appear at the same time as the fundrasing banner (the rule is one per page), and should also follow the docs for how banners are initiated: https://bedrock.readthedocs.io/en/latest/banners.html

Reply:
Is there a reason it should be different to usual top of the page banners and can't be dismissed? Usually the pencil banners can be closed (see the VPN banner as another example) https://www.mozilla.org/en-US/products/vpn/

Reply:
Good point - I'll check in with stakeholders and ask. Thanks for bringing this up